src/github.com/getlantern/flashlight/autoupdate.go

```diff
@@ -0,0 +1,6 @@
+package main
+
+var (
+	packageVersion   = "2.0.0-beta2"
```

Comment:
Yes. Maybe we should not rely on doing this everytime, this kind of thing could be more easily done by one of our build scripts, in fact I'll add this to my tasks on the build scripts front.